### PR TITLE
basic swig lexer

### PR DESCRIPTION
**Pull request by steevhise**
Basically a copy and tweak of the handlebars lexer.  This is only the basics, doesn't do as much with arguments and stuff that the Handlebars lexer did, so maybe you don't want to accept this. But for my purposes, it's enough. I don't have time at the moment to have it handle all the possible things you can do with the translation function within a swig template, but I thought it might be useful to at least have this back to you. thanks.

### CODE REVIEW

File: src/parser.js [hunk 1]
```diff
@@ -22,9 +25,10 @@ const lexers = {
 
   default: ['JavascriptLexer']
 }
-
+console.log('lexers are:' , lexers);
```
Comment (karellm) [inline src/parser.js]: Let's remove this guy!

File: src/lexers/swig-lexer.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      '(' +
+        '(?:' +
+          BaseLexer.variablePattern +
+          '(?:=' + BaseLexer.stringOrVariablePattern + ')?' +
+      '\\)' +
+      ')' +
+        '|' +
+        BaseLexer.stringPattern +
+      ')?'
+    // console.log(pattern);
```
Comment (karellm) [inline src/lexers/swig-lexer.js]: Let's remove this guy!

**Comment by steevhise** [comment]
Ok I'll probably be able to work more on it this week. Thanks.

**Comment by karellm** [comment]
@steevhise I'm closing this by lack of activity. Feel free to reopen if you ever get to work on it again.

**Pull request closed without merging**